server/plugin.go

```diff
@@ -20,9 +23,53 @@ type Plugin struct {
 	configuration *configuration
 }
 
-// ServeHTTP demonstrates a plugin that handles HTTP requests by greeting the world.
+// ServeHTTP demonstrates a plugin that handles HTTP requests.
 func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
We should have different endpoints instead of checking/adding the cookie to all requests made to the plugin since we're going to want to be able to make other requests to the plugin to mark groups/users as canaries. So for cookie checking we should use a path like `/api/v1/check`. See https://github.com/mattermost/mattermost-plugin-cloud/blob/master/server/api.go#L11 for examples

Reply:
/api/v1/check added and separate function created to handle checks.